<ui version="4.0" >
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Tiled</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionRecentFiles" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutQt" />
   </widget>
   <widget class="QMenu" name="menuMap" >
    <property name="title" >
     <string>Map</string>
    </property>
    <addaction name="actionResize" />
    <addaction name="actionProperties" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuMap" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <property name="windowTitle" >
    <string>Main toolbar</string>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen" />
   <addaction name="actionSave" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <action name="actionOpen" >
   <property name="icon" >
    <iconset>
     <normaloff>:/file/images/document-open.png</normaloff>:/file/images/document-open.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset>
     <normaloff>:/file/images/document-save.png</normaloff>:/file/images/document-save.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save...</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;About Tiled</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionResize" >
   <property name="icon" >
    <iconset>
     <normaloff>:/file/images/document-resize.png</normaloff>:/file/images/document-resize.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Resize</string>
   </property>
  </action>
  <action name="actionProperties" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>:/file/images/document-properties.png</normaloff>:/file/images/document-properties.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Properties</string>
   </property>
  </action>
  <action name="actionRecentFiles" >
   <property name="icon" >
    <iconset>
     <normaloff>:/file/images/document-open-recent.png</normaloff>:/file/images/document-open-recent.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Recent Files</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources>
  <include location="tiled.qrc" />
 </resources>
 <connections/>
</ui>
